styles.css

```diff
@@ -0,0 +1,4 @@
+.border-3 {
```

Comment:
Nah. Don't override library classes. I don't think you should go the thick border route anyway.